src/docs/integrations/slack/index.mdx

```diff
@@ -0,0 +1,89 @@
+---
+title: "Slack Integration"
+---
+
+Based off of this [post](https://forum.sentry.io/t/how-to-configure-slack-in-your-on-prem-sentry/3463)
+
+## Create a Slack App
+
+To use Sentry’s new Slack integration you’ll need to create an app in Slack bot app. Navigate to __Your Apps__ and click __Create new App__.
+
+![Create](./create.png)
+
+After naming your app and connecting your workspace, navigate to __Basic Information__. Here you’ll find your Client ID, Client Secret, and Verification token that lets your app access the Slack API.
+
+Here’s where you’ll connect your on-premise Sentry instance to your newly created Slack app.
+<dl>
+<dt><code>system.admin-email</code></dt>
+Declared in <code>config.yml</code>.
+
+<dd>
+<markdown>
+Copy your Client ID, Client Secret, and Verification Token.
+
+Set slack.legacy-app to False unless you are supporting an old bot integration (Pre June 2020)
+
+```yaml
+slack.client-id: <'client id'>
+slack.client-secret: <client secret>
+slack.verification-token: <verification token>
```

Comment:
@phyzical So we should actually now be telling people to use the `slack.signing-secret` instead of `slack.verification-token` since it's more secure. It's something that we added recently. Do you mind making this change?
 
![Screen Shot 2020-07-08 at 2 33 18 PM](https://user-images.githubusercontent.com/8533851/86972652-20b7f200-c128-11ea-8884-cc383d6798f5.png)

Reply:
hmm i did not notice that deprecation warning. i will give it a test later today if i get a chance then update the docs accordingly

Reply:
i gave it a quick test, seemed to sync the integration again, so updated docs accordingly